chef_master/source/dk_windows.rst

```diff
@@ -1,101 +1,195 @@
 
-.. end_tag
-
-Installing ChefDK on Windows
+Chef Workstation Supported Platforms
+-----------------------------------------------------
 
-Configure Environment
+Supported Host Operating Systems:
+
+.. list-table:: Supported Host Operating System
+   :widths: 15 15 
+   :header-rows: 1
+  
+   * - Platform
+     - Version
+   * - Apple macOS
+     - 10.11, 10.12, 10.13
+   * - Microsoft Windows
+     - 10, Server 2008 R2, Server 2012, Server 2012 R2, Server 2016
+   * - Red Hat Enterprise Linux / CentOS
+     - 6.x, 7.x
+   * - SUSE Enterprise Linux Server
+     - 11 SP4, 12 SP1+
+   * - Ubuntu 
+     - 14.04, 16.04, 18.04
+   * - Debian
+     - 7.x, 8.x, 9.x
+
+.. list-table:: Supported Target Operating System
+   :widths: 15 15 
+   :header-rows: 1
+
+   * - Platform
+     - Version
+   * - Microsoft Windows
+     - 10, Server 2008 R2, Server 2012, Server 2012 R2, Server 2016
+   * - Red Hat Enterprise Linux
+     - 6.x, 7.x
+   * - SUSE Enterprise Linux Server
+     - 11 SP4, 12 SP1+
+   * - Ubuntu
+     - 14.04, 16.04, 18.04
+   * - Debian
+     - 7.x, 8.x, 9.x
+
+Chef Workstation System Requirements
+----------------------------------------------------
+
+Minimum system requirements:
+
+* RAM: 2GB
+* Disk: 4GB
+* Running minimum settings may limit your ability to take advantage of Chef
+  Workstation tools such as Test Kitchen which creates and manages virtualized
+  test environments.
+
+Recommended system requirements:
+
+* RAM: 4GB
+* Disk 8GB
+
+Chef Workstation App
 -----------------------------------------------------
-.. tag ruby_set_system_ruby_as_chefdk_ruby_windows
 
-You can use the ``chef shell-init`` command with Windows PowerShell to add the appropriate variables to your environment.
+* Windows: No additional requirements
+* Mac: No additional requirements
+* Linux:
+   - You must have a graphical window manager running
+   - Additional libraries may be required. See `Running the Chef Workstation App <https://chef-workstation-app.html>`_ for more details.
 
-To try it in your current session:
+Installing ChefDK
+==================================================
 
-.. code-block:: bash
+In some cases, such as if you're working with older or unsupported systems, you may need to use ChefDK instead of Chef Workstation. Download ChefDK by following the installation instructions on `Installing ChefDK </install_dk.html>`_.
 
-   chef shell-init powershell | Invoke-Expression
+#. Visit the `ChefDK downloads page <https://downloads.chef.io/chefdk>`_ and select the appropriate package for your Windows version. Click on the **Download** button.
+#. Follow the steps to accept the license and install the Chef development kit. You will have the option to change your install location; by default the installer uses the ``C:\opscode\chefdk\`` directory.
+#. **Optional:** Set the default shell. On Microsoft Windows it is strongly recommended to use Windows PowerShell and ``cmd.exe``.
```

Comment:
I've never really understood this line.  It might be worth just removing it.

Reply:
OK to remove the section?